packages/tools/src/tools/segmentation/strategies/fillSphere.ts

```diff
@@ -44,7 +46,23 @@ function fillSphere(
     viewport as Types.IVolumeViewport
   );
 
-  triggerSegmentationDataModified(segmentationId);
+  const zMultiple = dimensions[0] * dimensions[1];
+  const minSlice = Math.floor(scalarIndex[0] / zMultiple);
+  const maxSlice = Math.floor(scalarIndex[scalarIndex.length - 1] / zMultiple);
+  const testArray = Array.from(
```

Comment:
rename

Reply:
testArray has been renamed to sliceArray